project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/components/box/error-page.less

```diff
@@ -3,25 +3,11 @@
     align-items: center;
     flex-wrap: wrap;
 
-    .is-no-flex & {
-        display: block;
-        .clearfix();
-    }
-
     &__text {
-        flex: 0 100%;
-
-        .is-no-flex & {
-            width: 100%;
-        }
+        width: 100%;
 
         @media @query-md {
-            flex: 1;
-
-            .is-no-flex & {
-                float: left;
-                width: 100%/2;
-            }
+            width: 100%/2;
```

Comment:
This is the same case as the comment above. Please check if this is a correct solution.

Reply:
Flex: 1 sets all flexible items to be same lenght. Is not necessary bcs, we set calculated width which will be same for both sides 50%

Reply:
It is okay for now. We will refactor this later.